Shared/mods/deathmatch/logic/luadefs/CLuaElementDefsShared.cpp

```diff
@@ -50,6 +50,41 @@ int CLuaElementDefs::GetElementData(lua_State* luaVM)
     return 1;
 }
 
+int CLuaElementDefs::GetAllElementData(lua_State* luaVM)
+{
+    //  table getAllElementData ( element theElement )
+
+#ifdef MTA_CLIENT
+    CClientEntity* pElement;
+#else
+    CElement* pElement;
+#endif
+
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData(pElement);
+
+    if (!argStream.HasErrors())
+    {
+        CLuaArguments Args;
+#ifdef MTA_CLIENT
+        CLuaArguments* pVariable = pElement->GetAllCustomData(&Args);
+#else
+        CLuaArguments* pVariable = CStaticFunctionDefinitions::GetAllElementData(pElement, &Args);
```

Comment:
You can just get rid of `CStaticFunctionDefinitions::GetAllElementData` server-side. It's only used here and is defined as this (below), which is essentially the same thing:

https://github.com/multitheftauto/mtasa-blue/blob/6941e0b0700a6d8c2775cc65de8022496b55e4d4/Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp#L403-L409